packages/nhost_sdk/lib/src/functions.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:http/http.dart' as http;
```

Comment:
Is dart supported by nhost now in addition to js/ts? How are endpoints specified?

Reply:
No, no Dart at the moment, although I'd definitely file a request over at https://github.com/nhost/nhost

Although sparse, the [docs here](https://docs.nhost.io/platform/serverless-functions) describe how the process works for JS/TS (it looks like there may be Go support in there as well? but I haven't tried it out)

Reply:
Got it @shyndman, then what is the purpose of this function? I misunderstood as serverless functions that are provided by nhost

Reply:
This is a client for calling serverless functions.

Maybe I need a better name?

Reply:
Ah understood. Maybe name could be else but i think documentation could be enough as well. I was looking at it without documentation hence got bit confused.

A question i have is why is it always making a post request? https://github.com/nhost/nhost-dart/pull/50/files/f0d0056b3e295ec7add49a99dcf933156682b0e6#diff-4be4a0d206d165857c3d6c4f69537dd96a82bf1da61223ba714539337db95fe2R34

I believe the serverless functions right now support both get and post. Wouldnt this cause inconsistency between the sdk and the nhost serverless function support?

Reply:
I'm actually just doing the same thing as the JS client: https://github.com/nhost/nhost-js/blob/main/src/clients/functions.ts

That said, you do make a good point. It may be worth adding a GET.

Reply:
Want to file it as a feature request? I'll get to it tomorrow.

Reply:
Yes I can do that. Also with v2 i dont know if things have changed and now the serverless function routes are only supported for POST. I know previously it used to support POST, GET and possibly DELETE as well. So maybe worth checking with Nhost team if that still is the case. 

Also I dont think i need that feature right now, i was just going through the code to understand more about V2 and saw that.

Reply:
Looks like all verbs are supported: https://github.com/nhost/cli/blob/2b01ba4c326c766b939b971ac20accc24f86e60c/functions/functions.go#L78

Reply:
Hmm, then might sense to have that support here as well for 2 reasons i believe:
- Consistency in the functionality supported by infrastructure and sdk
- Based on principle of least surprises, the user would be aware what kind of request is being made instead of automatic post request everytime

Created https://github.com/nhost/nhost-dart/issues/55 as well

Reply:
Thank you